.travis-build.sh

```diff
@@ -48,11 +48,14 @@ fi
 # subsequent command to build it except to test building it.
 
 set -e
+# Checks for the existence of a repository
+GITEXISTS="wget -q --spider"
```

Comment:
Instead of duplicating this definition here, is the one from `.travis-build-without-test.sh` visible?
Maybe `export` the variable there?

Reply:
Ping.